gammapy/modeling/models/spatial.py

```diff
@@ -1311,6 +1311,7 @@ class PiecewiseNormSpatialModel(SpatialModel):
 
     def __init__(self, coords, norms=None, interp="lin", **kwargs):
         self._coords = coords
+        self._coords["lon"] = Angle(self._coords.lon).wrap_at(180 * u.deg)
```

Comment:
this will move the problem at the anticenter, the wrap_angle could be computed such as is is always outside the range of the input coords maybe
self._wrap_angle = (coords.lon.max()-coords.lon.min())/2

Reply:
Okay, that is a good idea